src/app/core/navBarDropdown.service.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable()
+export class NavbarDropdownService {
+
+    constructor(
+        private route: ActivatedRoute
+    ) {}
+    //mobile or desktop
+    public isDesktop = new BehaviorSubject<boolean>(true);
+    //user options main navbar
+    public userDropdown = new BehaviorSubject<boolean>(false);
+
+    public levelsDropdown = new BehaviorSubject<boolean>(false);
+    public topicDropdown = new BehaviorSubject<boolean>(false);
+    public collectionsDropdown = new BehaviorSubject<boolean>(false);
+    public resourcesDropdown = new BehaviorSubject<boolean>(false);
+    //mobile slideouts
+    public isMHamburger = new BehaviorSubject<boolean>(false);
+    public isMSearch = new BehaviorSubject<boolean>(false);
+
+    externalResources = [
+    {name: 'CAE Resource Directory', link: 'http://www.caeresource.directory'},
+    {name: 'CAE Community Site', link: 'https://www.caecommunity.org/'}
+    ];
+    topics = ['topic 1', 'topic 2'];
```

Comment:
There is a topics route. If you do a search /topics in the routes.ts file in the environment folder you'll find it there.